config/passport.js

```diff
@@ -0,0 +1,33 @@
+const LocalStrategy = require('passport-local').Strategy;
+
+const User = require('../db/models/User');
+
+module.exports = (passport) => {
+  // used to serialize the user for the session
+  passport.serializeUser((user, done) => {
+    done(null, user.id);
+  });
+
+  // used to deserialize the user
+  passport.deserializeUser((id, done) => {
+    User.findById(id, (err, user) => {
+      done(err, user);
+    });
+  });
+
+  passport.use('local-signup', new LocalStrategy({
+
+    usernameField: 'email',
+    passwordField: 'password',
+    passReqToCallback: true,
+  },
+  (req, email, password, done) => {
+    process.nextTick(() => {
+      User.findOne({ email }, (err, user) => {
```

Comment:
move to backendQuery folders in order to reuse the same methods

Reply:
done